contracts/LoanManager.sol

```diff
@@ -100,14 +100,11 @@ contract LoanManager is Restricted {
 
         // calculate loan values based on ETH sent in with Tx
         uint tokenValue = rates.convertFromWei(augmintToken.peggedSymbol(), msg.value);
-        uint repaymentAmount = tokenValue.mul(products[productId].collateralRatio).roundedDiv(100000000);
-        repaymentAmount = repaymentAmount * 100;    // rounding 4 decimals value to 2 decimals.
-                                                    // no safe mul needed b/c of prev divide
-
-        uint mul = products[productId].collateralRatio.mul(products[productId].discountRate) / 1000000;
-        uint loanAmount = tokenValue.mul(mul).roundedDiv(100000000);
-        loanAmount = loanAmount * 100;  // rounding 4 decimals value to 2 decimals.
-                                        // no safe mul needed b/c of prev divide
+        uint repaymentAmount = tokenValue.mul(products[productId].collateralRatio).roundedDiv(1000000);
+
+        uint loanAmount = tokenValue.mul(products[productId].collateralRatio)
+                                    .mul(products[productId].discountRate)
+                                    .roundedDiv(1000000 * 1000000);
```

Comment:
we could do:
```
uint loanAmount = tokenValue.mul(products[productId].collateralRatio)
                                                   .roundedDiv(1000000);
                                                   .mul(products[productId].discountRate)
                                                   .roundedDiv(1000000);
```
so that we could have more potential scale but the double rounding might be tricky to recalc backwards on UI. Also if we are going to change token to uint64 once it won't be problem

Reply:
That's fine.

Minor comment: It might be cleaner/cheaper to lookup `products[productId]` only once and assign to a local variable.